src/app/showcase/chart-showcase/chart-showcase.component.html

```diff
@@ -12,13 +12,17 @@ <h4>Properties:</h4>
       <tbody>
         <tr>
           <td><code>height</code></td>
-          <td>The height of the chart in px for web and dp for native</td>
-          <td><code>number</code></td>
+          <td>
+            If a number, the height is given in pixels. If given a percentage string (for example
```

Comment:
Does this also allow a pixel value, e.g. `300px` and/or `em`s? If yes, please add to description.

Reply:
I haven't changed anything to the existing interface.